src/Sulu/Bundle/ContentBundle/Controller/PreviewController.php

```diff
@@ -149,8 +149,9 @@ public function renderAction(Request $request, $contentUuid)
                             'language' => $locale
                         )
                     ),
-                'wsUrl' => 'ws://' . $request->getHttpHost(),
-                'wsPort' => $this->container->getParameter('sulu_content.preview.websocket.port'),
+                'wsUrl' => 'ws://' . $this->container->getParameter('sulu_websocket.server.http_host'),
+                'wsPort' => $this->container->getParameter('sulu_websocket.server.port'),
+                'wsRoute' => '/content/preview',
```

Comment:
Wouldn't you make this configureable too? Otherwise it could be hard to handle naming conflicts. However, this configuration should probably be optional.